src/js/containers/award/financialAssistance/CFDAVizContainer.jsx

```diff
@@ -64,6 +64,11 @@ export default class CFDAVizContainer extends React.Component {
         this.setState({ previousView: 'tree', view: 'single', cfda });
     }
 
+    onDropdownClick = (title) => {
```

Comment:
Would be nice to have a test for this.

Reply:
Yeah, this ultimately should change and the Picker should allow for value but until https://github.com/fedspendingtransparency/data-transparency-ui/pull/29 gets merged...